src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -106,7 +106,7 @@ export const ItemNav = <D,>({ info, config, additionalData, onDelete, onStart, o
 
     const MAX_STACK = 4;
 
-    const additionalPreviewComp = (additionalPreview && additionalPreview(additionalData, curItemID));
+    const additionalPreviewComp = (additionalPreview && !!additionalData && additionalPreview(additionalData, curItemID));
```

Comment:
Did you just `!!`?

Reply:
There was a `0` showing up in the bottom right of the preview if there was no `!!`

Reply:
Cause it's an enum.. right alright fine I accept